doc/developer-guide/architecture/cache-store.en.rst

```diff
@@ -1,83 +0,0 @@
-.. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Intersting, I can not find the commit details where is says that this file is going to get deleted.